_episodes/05-exporting-data-2.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Exporting data"
+teaching: 10
+exercises: 0
+questions:
+- "How can we export data from spreadsheets in a way that is useful for downstream applications?"
+objectives:
+- "Store spreadsheet data in universal file formats."
+- "Export data from a spreadsheet to a CSV file."
+keypoints:
+- "Data stored in common spreadsheet formats will often not be read correctly into data analysis software, introducing errors into your data."
+- "Exporting data from spreadsheets to formats like CSV or TSV puts it in a format that can be used consistently by most programs."
+---
+
+Authors:**Christie Bahlai**, **Aleksandra Pawlik**<br>
```

Comment:
remove authors here as well

Reply:
Yep. Will do when I put in a PR for those episodes.